internal/support/notifications/application/subscription.go

```diff
@@ -137,7 +137,11 @@ func DeleteSubscriptionByName(name string, ctx context.Context, dic *di.Containe
 		return errors.NewCommonEdgeX(errors.KindContractInvalid, "name is empty", nil)
 	}
 	dbClient := container.DBClientFrom(dic.Get)
-	err := dbClient.DeleteSubscriptionByName(name)
+	_, err := dbClient.SubscriptionByName(name)
+	if err != nil {
+		return errors.NewCommonEdgeXWrapper(err)
+	}
```

Comment:
move this check to the application level

Reply:
This is already the application level.